Mu2eG4/src/Mu2eG4_module.cc

```diff
@@ -451,9 +451,9 @@ namespace mu2e {
 
 
   void Mu2eG4::endJob(){
-
     if ( _exportPDTEnd ) exportG4PDT( "End:" );
     _physVolHelper.endRun();
+    delete _runManager;
```

Comment:
I think you could accomplish the same thing by keeping it a unique_ptr and adding one line here:

_runManager->release(nullptr);

or something like that.  I just don't like leaving bare pointers around as a model for others to follow.

Reply:
I am not sure why replacing pointers works. I view it as a temporary solution to remove warnings and I do not like it either.
One thing which it does is defining the exact point where the run manager is deleted. 
The release does not do it: https://www.cplusplus.com/reference/memory/unique_ptr/release/

Reply:
Adding my previous reply into the thread, where it belongs: Oops, reset() not release.

Reply:
Is the real reason that it works the removal of Clean() and the change of pointer type has nothing to do with it?

Reply:
Sorry, I missed/misread reset; that should do it. I change it to it and see if it still works. 
The change (warnings going away) comes from that RunMamager deletion, not removal of Clean(); I double check it though.

Reply:
I agree with Rob that keeping the unique_ptr is preferable to switching to a raw pointer.  They have the same functionality but unique_ptr is less error prone.